geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -803,7 +803,7 @@ public PartitionedRegion(String regionName, RegionAttributes regionAttributes,
     if (this.getEvictionAttributes() != null
         && this.getEvictionAttributes().getAlgorithm().isLRUHeap()) {
       this.sortedBuckets = new ArrayList<BucketRegion>();
-      this.bucketSorter = LoggingExecutors.newScheduledThreadPool("BucketSorterThread", 1);
+      this.bucketSorter = LoggingExecutors.newScheduledThreadPool("BucketSorterRunnable", 1);
```

Comment:
This one should not be "BucketSorterRunnable". This is the thread name so the original "BucketSorterThread" was fine.

Reply:
I unit test would be great but not required.

Reply:
added the unit test and renamed the thread